### ISSUE TITLE
Add Facebook and Twitter header tags

### ISSUE THREAD

**Issue by iamdoubz** [PrivateBin/PrivateBin#635]
This is probably a feature request, but for now I'm asking where to make the edits. I want to add the `og:` et al. header tags as well as `twitter:` et al. tags. I'm not saying that I dislike the PrivateBin logo, I just like to use my own and have a more complete header meta tags.

### Example additions:
```html
	<meta property="og:title" content="txt"/>
	<meta property="og:url" content="https://txt.dou.bet/"/>
	<meta property="og:description" content="A pastebin service"/>
	<meta property="article:published_time" content="2020-06-11T10:55:00-05:00"/>
	<meta property="og:site_name" content="Dou.Bet"/>
	<meta property="og:image" content="https://txt.dou.bet/img/txtfb.png"/>
	<meta property="og:image:width" content="5000"/>
	<meta property="og:image:height" content="2500"/>
	<meta property="og:locale" content="en_US"/>
	<meta name="twitter:text:title" content="txt"/>
	<meta name="twitter:image" content="https://txt.dou.bet/img/txtfb.png"/>
	<meta name="twitter:card" content="summary_large_image"/>
```

## Basic information

**Server address**: https://txt.dou.bet

**Server OS:** Ubuntu 18.04

**Webserver:** Apache 2.4.29

**Browser:** Chrome 83.0.4103.97 and Edge 83.0.478.45

**PrivateBin version:** 1.3.4

I can reproduce this issue on <https://privatebin.net>:
  - [x] Yes
  - [ ] No

**Comment by elrido** [comment]
I think this already got implemented in #593

**Comment by iamdoubz** [comment]
Okay. I did the `.tar.gz` release instead of grabbing it from master. But, your link helped me find where I needed to go: `tpl/bootstrap.php`

Thanks!
